### PR TITLE
initial project structure

### PR DESCRIPTION
**Pull request by sbalay**
## Summary
Create initial components, services, routes and redux actions/reducers.
Api integration

## Screenshots
screen shot 2017-08-22 at 10 32 09 am
screen shot 2017-08-22 at 10 32 02 am

## Trello Card
https://trello.com/c/GR6K1UsJ/1-muestra-cada-board-existente
https://trello.com/c/sE8c36XD/18-json-server
https://trello.com/c/malKAyOo/22-detalle-de-board

### CODE REVIEW

File: src/redux/boards/actions.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+export const actions = stringArrayToObject([
+  'GET_BOARDS',
+  'GET_BOARDS_SUCCESS',
+  'GET_BOARDS_FAILURE',
+  'GET_BOARD',
+  'GET_BOARD_SUCCESS',
+  'GET_BOARD_FAILURE'
+]);
+
+const actionCreators = {
```
Comment (mvbattan) [inline src/redux/boards/actions.js:13]: Should we show https://github.com/infinitered/reduxsauce ? ☺️

File: src/app/screens/Board/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    );
+  }
+}
+
+const mapStateToProps = store => ({
+  loading: store.boards.loading,
+  board: store.boards.detail
+});
+
+export default connect(mapStateToProps)(BoardDetailContainer);
```
Comment (mvbattan) [inline src/app/screens/Board/index.js]: Place `export default` at the bottom of this file

File: src/app/screens/BoardsList/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          )}
+        </ul>;
+  }
+}
+
+const mapStateToProps = store => ({
+  boards: store.boards.list
+});
+
+export default connect(mapStateToProps)(BoardsListContainer);
```
Comment (mvbattan) [inline src/app/screens/BoardsList/index.js]: Place `export default` at the bottom of this file

**Review by mvbattan — CHANGES_REQUESTED** [review]
Small changes 😊

File: src/redux/boards/actions.js (hunk 1 above)
Comment (sbalay) [reply src/redux/boards/actions.js:13]: We should! I created this card in trello https://trello.com/c/B970IoDy/23-chequear-redux-sauce

File: src/config/routes.js [hunk 4]
```diff
@@ -3,15 +3,20 @@ import { ConnectedRouter } from 'react-router-redux';
...
+};
 
 function AppRoutes() {
   return (
     <ConnectedRouter history={history}>
       <div>
-        <Route exact path="/" component={App} />
-        <Route path="/dummy" component={Dummy} />
+        <Route exact path="/" component={BoardsList} />
+        <Route exact path="/board/:id" component={Board} />
```
Comment (mvbattan) [inline src/config/routes.js:19]: Shouldn't these go to Route constants ?

**Review by mvbattan — APPROVED** [review]
@sbalay Looks good to me, just leaving a single comment

**Review by IgnacioAbadie — APPROVED** [review]

File: src/config/routes.js (hunk 4 above)
Comment (sbalay) [reply src/config/routes.js:19]: Yep

**Pull request merged**